panda-market/src/shared/api/articles/getBoards.ts

```diff
@@ -0,0 +1,36 @@
+import { AxiosError, AxiosResponse } from 'axios';
+import { ArticleResponse, GetArticleRequest } from '@/entities/articles';
+import { instance } from '@/shared/api';
+import { API_PATH } from '@/shared/config';
+
+export const getBoards = async ({
+  page = 1,
+  orderBy = 'recent',
+  pageSize = 10,
+  keyword,
+}: GetArticleRequest) => {
+  return await instance
+    .get<GetArticleRequest, AxiosResponse<ArticleResponse>>(API_PATH.article, {
+      params: {
+        page,
+        orderBy,
+        pageSize,
+        keyword,
+      },
+    })
+    .then((res) => {
+      // if (res.status === 200) {
+      //   return res.data;
+      // }
+
+      // if (res.status === 404) {
+      //   throw new Error(
+      //     `요청한 데이터를 찾을 수 없습니다. \n ErrorStatus: ${res.status} \n ErrorMessage: ${res.data}`,
+      //   );
+      // }
+      return res.data;
+    })
+    .catch((e) => {
+      return e;
```

Comment:
```suggestion
      throw e;
```

에러를 던져서 `getBoards`를 사용하는 측에서 에러를 처리 할 수 있도록 합시다. 
react-query에서 에러 상태를 잘 캐치 해줄 것 같네요.